### DISCUSSION TITLE
how to find rejected node?

### DISCUSSION THREAD

**Discussion by eleaner** [gravitational/teleport#48555] [Q&A]
Hi

my cluster complains about rejected node with 
`Connection from Node 0dc0e352-xxxx-xxxx-xxxx-xxx.teleport.tld at 172.16.0.5:49990, running an unsupported version of v14.3.32 was rejected. Connections will be allowed after upgrading the agent to v15.0.0 or newer`

it drives me crazy because I have no idea what the hostname it and how to find it
172.16.0.0/24 looks like docker network (possibly)

Any ideas?
can I remove it from the cluster altogether?

**Comment by webvictim** [comment]
You'll need to terminate the Teleport agent process running on the node to stop it rejoining the cluster for good. If your Teleport cluster is running in Docker, then you're probably seeing the Docker NAT IP rather than the node's true IP.

If you temporarily set the variable `TELEPORT_UNSTABLE_ALLOW_OLD_CLIENTS=yes` in the Teleport cluster's environment and restart the server, the rogue agent should then be allowed to connect. You can then see this agent's UUID in `tsh ls -v` and figure out its hostname, which should then tell you where it is and you can go on to upgrade/remove it.
